src/components/GameItem.js

```diff
@@ -4,23 +4,59 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  // this is so we can get setstate right?
```

Comment:
Yes! We change it to a class than inherits from Component because it can hold state.